api/src/main/java/io/opentelemetry/trace/Status.java

```diff
@@ -44,163 +43,18 @@
-     * applied to a non-directory, etc.
-     *
-     * <p>A litmus test that may help a service implementor in deciding between FAILED_PRECONDITION,
-     * ABORTED, and UNAVAILABLE: (a) Use UNAVAILABLE if the client can retry just the failing call.
-     * (b) Use ABORTED if the client should retry at a higher-level (e.g., restarting a
-     * read-modify-write sequence). (c) Use FAILED_PRECONDITION if the client should not retry until
-     * the system state has been explicitly fixed. E.g., if an "rmdir" fails because the directory
-     * is non-empty, FAILED_PRECONDITION should be returned since the client should not retry unless
-     * they have first fixed up the directory by deleting files from it.
-     *
-     * @since 0.1.0
-     */
-    FAILED_PRECONDITION(9),
-
-    /**
-     * The operation was aborted, typically due to a concurrency issue like sequencer check
-     * failures, transaction aborts, etc.
-     *
-     * <p>See litmus test above for deciding between FAILED_PRECONDITION, ABORTED, and UNAVAILABLE.
-     *
-     * @since 0.1.0
-     */
-    ABORTED(10),
 
     /**
-     * Operation was attempted past the valid range. E.g., seeking or reading past end of file.
-     *
-     * <p>Unlike INVALID_ARGUMENT, this error indicates a problem that may be fixed if the system
-     * state changes. For example, a 32-bit file system will generate INVALID_ARGUMENT if asked to
-     * read at an offset that is not in the range [0,2^32-1], but it will generate OUT_OF_RANGE if
-     * asked to read from an offset past the current file size.
-     *
-     * <p>There is a fair bit of overlap between FAILED_PRECONDITION and OUT_OF_RANGE. We recommend
-     * using OUT_OF_RANGE (the more specific error) when it applies so that callers who are
-     * iterating through a space can easily look for an OUT_OF_RANGE error to detect when they are
-     * done.
-     *
-     * @since 0.1.0
+     * The operation has been validated by an Application developers or Operator to have completed
+     * successfully.
      */
-    OUT_OF_RANGE(11),
-
-    /**
-     * Operation is not implemented or not supported/enabled in this service.
-     *
-     * @since 0.1.0
-     */
-    UNIMPLEMENTED(12),
-
-    /**
-     * Internal errors. Means some invariants expected by underlying system has been broken. If you
-     * see one of these errors, something is very broken.
-     *
-     * @since 0.1.0
-     */
-    INTERNAL(13),
-
-    /**
-     * The service is currently unavailable. This is a most likely a transient condition and may be
-     * corrected by retrying with a backoff.
-     *
-     * <p>See litmus test above for deciding between FAILED_PRECONDITION, ABORTED, and UNAVAILABLE.
-     *
-     * @since 0.1.0
-     */
-    UNAVAILABLE(14),
+    OK(0),
 
-    /**
-     * Unrecoverable data loss or corruption.
-     *
-     * @since 0.1.0
-     */
-    DATA_LOSS(15),
+    /** The default status. */
+    UNSET(17),
```

Comment:
How did we come with this numbers? See https://github.com/open-telemetry/opentelemetry-proto/pull/224/files and maybe we should follow the same numbers so translation to otlp is just direct mapping.

Reply:
I made them up. I was planning on fixing once the proto change was merged.

Reply:
I'll update this PR to match what is in the proto PR, now that it looks like there aren't any more changes happening.

Reply:
I think the numbers are now matching the deprecated code, but we should update to 0, 1, 2 here I believe.

Reply:
yes. will do...that was my intent. :)

Reply:
done